dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -76,7 +76,6 @@ def __init__(self, options: NumericalOptions = None) -> None:
         self.histogram_selection: str | None = None
         self.user_set_histogram_bin: int | None = None
         self.bias_correction: bool = True  # By default, we correct for bias
-        self._mode_is_enabled: bool = True
```

Comment:
Why is this removed?

Reply:
Weird I think this may solve some of my issue above. This should not be deleted